codewof/templates/users/group_detail.html

```diff
@@ -0,0 +1,205 @@
+{% extends "base.html" %}
+{% load static %}
+
+{% load i18n %}
+{% load crispy_forms_tags %}
+{% load user_extras %}
+{% load tz %}
+
+{% block title %}{% trans "New Group" %}{% endblock %}
+
+{% block content %}
+<div class="container">
+  {% if is_admin %}
+  <div id="alert-div" class="sticky-top" style="top: 40px">
+    <div id="update-success-alert" class="alert alert-success alert-dismissible collapse" role="alert">
+      Memberships successfully updated.
+    </div>
+
+    <div id="update-danger-alert" class="alert alert-danger alert-dismissible collapse" role="alert"></div>
+  </div>
+
+  <div id="demote-self-modal" class="modal" tabindex="-1" role="dialog">
+    <div class="modal-dialog" role="document">
+      <div class="modal-content">
+        <div class="modal-header">
+          <h5 class="modal-title">Warning!</h5>
+          <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+            <span aria-hidden="true">&times;</span>
+          </button>
+        </div>
+        <div class="modal-body">
+          <p>You are about to demote yourself. You will lose access to Admin privileges for this group.</p>
+
+          <p>Are you sure?</p>
+        </div>
+        <div class="modal-footer">
+          <button id="demote-self-modal-button" type="button" class="btn btn-primary">Continue</button>
+          <button type="button" class="btn btn-secondary" data-dismiss="modal">Close</button>
+        </div>
+      </div>
+    </div>
+  </div>
+  {% endif %}
+
+  <div class="row">
+    <div class="col-12 col-md-8">
+      <div class="row">
+        <div class="col mb-3">
+          <h1>{{ group.name }}</h1>
+          <p>{{ group.description }}</p>
+        </div>
+
+        <div class="col-sm-auto mb-3">
+          {% if is_admin %}
+          <a class="btn btn-primary btn-sm d-block mb-1 mt-2" href="{% url 'users:groups-edit' group.pk %}">
+            Edit Group</a>
+          <a class="btn btn-danger btn-sm d-block mb-1" href="{% url 'users:groups-delete' group.pk %}">Delete Group</a>
+          {% endif %}
+          <a id="leave-button" class="btn btn-warning btn-sm d-block mb-1 {% if not is_admin %} mt-2 {% endif %}"
+             {% if only_admin %} style="visibility: hidden" {% endif %}
+             href="{% url 'users:groups-memberships-delete' user_membership.pk %}">Leave Group</a>
```

Comment:
It will display for admins, but not if the user is the only admin of the group. This is to prevent issues of having a group with no admins such that the group cannot be managed by anyone. I still render it as the button may be set to visible or invisible when the user updates the membership table (e.g. removes or demotes the only other admin).

Reply:
Gotcha, it's the second part of your reply I hadn't considered.